sjb/config/test_cases/test_branch_openshift_ansible_logging.yml

```diff
@@ -19,33 +19,14 @@ extensions:
         fi
         hack/build-images.sh
     - type: "script"
-      title: "build an openshift-ansible release"
```

Comment:
So in general whenever possible we need to be building RPMs and using built releases for CI testing. The issues you saw with the `tito tag` command were entirely user error on the part of whoever branched Origin and OpenShift Ansible for the new release. We have doc for whoever is doing that branching that, if followed, will cause no errors on that end. Let's not remove this step.

Reply:
But is it absolutely necessary to do that here?  Because isn't building and installing ansible being done elsewhere, in other tasks?

Reply:
I would say yes, for the simple reason that the reason you took this out -- `tito` issues -- was an operator error and should not, in a perfect world, happen again. You incur no overhead with the RPM approach if we do our due diligence.

Reply:
That wasn't the only reason I took it out.  The primary reason is that it isn't necessary, and the secondary reason is that it has been fragile and caused problems in the past, the latest tito issue being just one of them.

Reply:
I guess I would challenge that -- why is it not necessary? How many of our customers are running playbooks from source? If we're not testing the build --> install --> use flow for these using RPMs we're not testing end user experience. Before we started doing this in our CI we had a slew of issues coming back from QE about this sort of thing.

How else has it been fragile? Of all things packaging the Ansible playbooks into an RPM has been the least fragile of all of our builds steps as far as I'm aware. If there are other specific concerns we should address those.

Reply:
I just want a way to run logging tests as simply as possible.

If we want to simulate the customer experience as much as possible, then we need a job that will install all of openshift + logging in one run of ansible-playbook.

Perhaps we need two sets of jobs - one for "lower" level logging integration testing, and one for "higher" level full installation path testing, which will install openshift + logging (and + metrics, and + whatever other options).

logging developers need a way to run the logging CI tests, locally or in an AWS devenv, in as close a way as possible to the way jenkins runs them.  Using `oct sync local` to sync local ansible and logging repos has caused a lot of grief related to building/installing ansible and tagging.  This is simply not necessary for 95% of developer and "lower" level logging CI workflow.